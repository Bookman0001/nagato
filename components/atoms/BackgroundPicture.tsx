import Image from 'next/image'

interface Props {
  src: string
  alt: string
}

export default function Background({ src, alt }: Props) {
  return (
    <>
      <Image src={src} alt={alt} width={1024} height={400} />
    </>
  )
}
