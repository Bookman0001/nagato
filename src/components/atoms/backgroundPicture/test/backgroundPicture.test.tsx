import { shallow } from 'enzyme'

import BackgroundPicture from '../'

test('BackgroundPicture', () => {
  const component = shallow(
    <BackgroundPicture alt={'test'} src={'https://placehold.jp/150x150.png'} />
  )
  expect(component).toMatchSnapshot()
})
