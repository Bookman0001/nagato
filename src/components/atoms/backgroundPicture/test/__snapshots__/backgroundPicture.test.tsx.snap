// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundPicture 1`] = `
<Fragment>
  <styled.img
    alt="test"
    height={400}
    src="https://placehold.jp/150x150.png"
    width={1024}
  />
</Fragment>
`;
